lib/index.js

```diff
@@ -12,23 +12,54 @@
       preflightContinue: false
     };
 
+  function varyHeadersOn(vary, headers) {
```

Comment:
It appends a vary header with the passed fields as value. It's easier when used like this:
```js
varyHeadersOn('Origin', headers); // intended to read as varyHeadersOnOrigin(headers)
```